Building for x86_64-unknown-linux-gnu fails

`cargo build` on an `x86_64-unknown-linux-gnu` target works correctly, but running `cargo build --target x86_64-unknown-linux-gnu` causes the following error:

```
/home/dario/.cargo/registry/src/github.com-1ecc6299db9ec823/mbedtls-sys-auto-2.18.4/vendor/crypto/library/aes.c:31:10: fatal error: '/home/dario/work/roche/server/target/x86_64-unknown-1-gnu/debug/build/mbedtls-sys-auto-9b28b4beb79db467/out/config.h' file not found
#include MBEDTLS_CONFIG_FILE
         ^
<command line>:1:29: note: expanded from here
#define MBEDTLS_CONFIG_FILE </home/dario/work/roche/server/target/x86_64-unknown-linux-gnu/debug/build/mbedtls-sys-auto-9b28b4beb79db467/out/config.h>
```
In particular, it tries to include a file from a `x86_64-unknown-1-gnu` target for some reason.
What is especially strange is that the error helpfully shows that the definition being expanded is in fact correct.

--- a/ct.sh
+++ b/ct.sh
@@ -11,7 +11,8 @@ fi
 
 if [ $TRAVIS_RUST_VERSION = "stable" ] || [ $TRAVIS_RUST_VERSION = "beta" ] || [ $TRAVIS_RUST_VERSION = "nightly" ]; then
     rustup default $TRAVIS_RUST_VERSION
-    cargo test
+    # make sure that explicitly providing the default target works
+    cargo test --target x86_64-unknown-linux-gnu
     cargo test --features spin_threading
     cargo test --features rust_threading
     cargo test --features zlib

--- a/mbedtls-sys/build/bindgen.rs
+++ b/mbedtls-sys/build/bindgen.rs
@@ -43,7 +43,7 @@ impl super::BuildConfig {
             .log(&logger)
             .clang_arg("-Dmbedtls_t_udbl=mbedtls_t_udbl;") // bindgen can't handle unused uint128
             .clang_arg(format!(
-                "-DMBEDTLS_CONFIG_FILE=<{}>",
+                "-DMBEDTLS_CONFIG_FILE=\"{}\"",
                 self.config_h.to_str().expect("config.h UTF-8 error")
             )).clang_arg(format!(
                 "-I{}",

--- a/mbedtls-sys/build/cmake.rs
+++ b/mbedtls-sys/build/cmake.rs
@@ -14,7 +14,7 @@ impl super::BuildConfig {
     pub fn cmake(&self) {
         let mut cmk = cmake::Config::new(&self.mbedtls_src);
         cmk.cflag(format!(
-            r#"-DMBEDTLS_CONFIG_FILE="<{}>""#,
+            r#"-DMBEDTLS_CONFIG_FILE="\"{}\"""#,
             self.config_h.to_str().expect("config.h UTF-8 error")
         ))
         .define("ENABLE_PROGRAMS", "OFF")

